src/pages/BlogPage.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axiosInstance from "../api/axios";
+import "./pageStyle/BlogPage.css";
+
+const BlogPage = () => {
+  const [displayBlogs, setDisplayBlogs] = useState([]);
+  const [page, setPage] = useState(1);
+  let navigate = useNavigate()
+
+  const getBlog = async () => {
+    try {
+      const { data } = await axiosInstance.get();
+      const allPost = data.getBlogPost.docs
+      setDisplayBlogs((prevPost) => [...prevPost, ...allPost]);
+      loadMore()
```

Comment:
After getting blogs, you load more blogs?

Reply:
its just a function for the page, I can use the state function. I t will be adjusted